### PR TITLE
[ARM/CI] Add ARM CI script for cross build

### PR DESCRIPTION
**Pull request by jyoungyun**
Implement ARM CI using Docker for arm and armel
Call the script in netci.groovy

Related issue : 
https://github.com/dotnet/core-setup/issues/790
https://github.com/dotnet/core-setup/issues/1400
https://github.com/dotnet/coreclr/issues/9273
https://github.com/dotnet/corefx/pull/15900

### CODE REVIEW

**Comment by hqueue** [comment]
Related PR for CoreCLR is available at https://github.com/dotnet/coreclr/pull/9445

**Comment by hqueue** [comment]
@dotnet-bot test ci please

**Comment by hqueue** [comment]
@dotnet-bot test Ubuntu16.04 arm Release Build
@dotnet-bot test Tizen4.0.0 armel Release Build

**Comment by hqueue** [comment]
@dotnet-bot skip ci test

File: scripts/arm32_ci_script.sh [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+        buildRootfsCmd="sudo ./cross/build-rootfs.sh armel tizen --skipunmount"
+    else
+        echo "ERROR: unknown buildArch $__buildArch"
+        exit 1
+    fi
+    $buildRootfsCmd
+    sudo chown -R $(id -u -n) cross/rootfs
+
+    # Cross building core-setup with rootfs in Docker
+    if [ "$__buildArch" == "arm" ]; then
```
Comment (gkhanna79) [inline scripts/arm32_ci_script.sh]: I presume this PR needs to be updated in a similar manner as the CoreFX Pr for the CI script, right?

Comment (jyoungyun) [inline scripts/arm32_ci_script.sh]: Yes, right. I will update this ASAP.

**Comment by jyoungyun** [comment]
@dotnet-bot help

**Comment by jyoungyun** [comment]
@dotnet-bot help

**Comment by jyoungyun** [comment]
@dotnet-bot test ci please
@dotnet-bot test Ubuntu arm Release 
@dotnet-bot test Ubuntu armel Release

**Comment by jyoungyun** [comment]
@dotnet-bot test ci please
@dotnet-bot test Ubuntu arm Release
@dotnet-bot test Ubuntu armel Release

**Comment by hseok-oh** [comment]
@dotnet-bot test ci please
@dotnet-bot test Ubuntu arm Release Build please
@dotnet-bot test Ubuntu armel Release Build please

**Comment by hseok-oh** [comment]
@jyoungyun @gkhanna79 It seems we cannot test newly defined job for arm/linux. If you want careful solution for CI machine, I suggest you to use step-by-step PR.

1. 1st PR: define new ARM/Linux job in netci.groovy that call scripts/arm32_ci_script.sh, but do nothing in script.
2. Next PR: modify scripts/arm32_ci_script.sh to build operation.

**Comment by jyoungyun** [comment]
@hseok-oh Even though I make a seperate patch like your suggestion, but I'm still not sure that my changes about netci.groovy is working well. Is it okay if this patch pass 'dotnet_core-setup/master/GenPRTest/generator_prtest' build alone?

**Comment by jyoungyun** [comment]
@dotnet-bot test ci please
@hseok-oh @gkhanna79 I modified this patch as following @hseok-oh 's comments. This patch is simply to add Ubuntu arm and Tizen armel CI job only and the arm32_ci script does nothing. PTAL

**Comment by jyoungyun** [comment]
@dotnet-bot test ci please

**Comment by jyoungyun** [comment]
@gkhanna79 @jashook @hqueue @hseok-oh And can I fix dotnet-ci like https://github.com/jyoungyun/dotnet-ci/blob/tizen/add_tizen_to_machine_map/jobs/generation/Utilities.groovy#L324L332 ? To use the Tizen os name, I must modify Jobs Utilities file.

**Comment by gkhanna79** [comment]
Change LGTM. Let me know when it is ready to merge @jyoungyun

**Comment by jyoungyun** [comment]
@gkhanna79 It's okay to merge this PR. Thank you.

**Comment by gkhanna79** [comment]
@jyoungyun I have merged this. Can you please trigger a dummy pr to confirm that Linux Arm32 CI build looks good?

**Comment by jyoungyun** [comment]
@gkhanna79 I made a new PR(#1572) for confirming that ARM CI works well.

**Pull request merged**